docs/SimpleFormIterator.md

```diff
@@ -70,6 +70,7 @@ const OrderEdit = () => (
 | Prop     | Required | Type           | Default               | Description                         |
 |----------|----------|----------------|-----------------------|-------------------------------------|
 | `addButton` | Optional | `ReactElement` | - | Component to render for the add button |
+| `clearButton` | Optional | `ReactElement` | - | Component to render for clear all items button |
```

Comment:
I don't see that prop used in the SimpleFormIterator code. Besides, we should allow a `false` value to disable it.

Reply:
Should we give individual `false` case for this?

I mean , should the user is able to disable clear array action by setting `disableRemove` prop to `true` or `arrayClear(example)` to `false`, for example?

Reply:
changed doc regarding request : [b24af8c](https://github.com/marmelab/react-admin/pull/8302/commits/b24af8ccb821196b478c4b516609a54e7503f826)

`clearButton`. is now `boolean` type, and the default value is `true`. User is able to disabled the clear array action by giving `false` to `clearButton` prop